src/App.js

```diff
@@ -1,17 +1,31 @@
-import logo from './logo.svg';
+import React from 'react';
+import "bootstrap/dist/css/bootstrap.min.css";
 import './App.css';
 import Header from './header.js';
 import Main from './main.js';
 import Footer from './footer.js';
+import beastData from "./data.json";
 
-function App() {
-  return (
-    <>
-      <Header />
-      <Main />
-      <Footer />
-   </>
-  );
+class App extends React.Component {
+
+  constructor(props){
+    super(props);
+    this.state = {
+      selectedBeast: null
+    }
+  }
+
+  // updateSelected = (newSelection) => {this.setState({selectedBeast: newSelection}); };
+
+  render() {
+    return (
+      <div>
+        <Header header="Horns" />
+        <Main beastData={beastData} updateSelected={this.updateSelected}/>
+        <Footer footer="Made by David, Brains by Hexx/Sara" />
+      </div>
```

Comment:
Here you are on your way as well! :)

You can:
- Create a `selectedBeast` component and use it here in App.js
- Use the state in the App to render an individual beast in a Modal in the `selectedBeast` component, you will need to make use of props. How can you tell selectedBeast it needs to know about the state of the modal?